### ISSUE TITLE
Bug happens when parsing `[[c] [d]'1]*1`

### ISSUE THREAD

**Issue by UlyssesZh** [alda-lang/alda-core#83]
```shell
alda parse -c "[[c] [d]'1]*1"
```
outputs
```
ERROR class clojure.lang.MapEntry cannot be cast to class java.lang.Number (clojure.lang.MapEntry is in unnamed module of loader 'app'; java.lang.Number is in module java.base of loader 'bootstrap')
```

**Comment by daveyarwood** [comment]
Thanks for reporting! I think this is a legit bug. Will look into it.

**Comment by daveyarwood** [comment]
Fixed in Alda 1.4.1!
